app/images/mm-info-icon.svg

```diff
@@ -1,3 +0,0 @@
-<svg viewBox="0 0 10 10" xmlns="http://www.w3.org/2000/svg">
-    <path d="M5 0C2.2 0 0 2.2 0 5s2.2 5 5 5 5-2.2 5-5-2.2-5-5-5zm0 2c.4 0 .7.3.7.7s-.3.7-.7.7-.7-.2-.7-.6.3-.8.7-.8zm.7 6H4.3V4.3h1.5V8z" fill="#b8b8b8"/>
-</svg>
```

Comment:
Not being used. No results found

![Screen Shot 2022-03-24 at 5 37 50 PM](https://user-images.githubusercontent.com/8112138/160032032-b11a7502-c749-47fb-afa8-eace9add29af.png)